### ISSUE TITLE
participle with cop: VERB or ADJ?

### ISSUE THREAD

**Issue by nschneid** [UniversalDependencies/UD_English-EWT#355]
There are a number of cases in both [EWT](http://universal.grew.fr/?custom=63307761d9938) and [GUM](http://universal.grew.fr/?custom=6330761a0556c) where a participial VERB functioning like an adjective is treated as a copular (`cop`) predicate. Is this correct? Should *be* be `aux` rather than `cop` for some of these, or VERB changed to ADJ?

This includes examples with compounding like

- all was **mouth-watering**
- ...is **mind-blowing**
- your cat is **stressed out**
- It's **tailor-made** for...

This is important for the implementation of outer subjects (#310). Another example matched by the above query:

- The most reliable way to study a conversational interjection is by examining cases of actual use. 
- An example of this would be studying for an exam. 

Here, "way" should be an outer subject of "examining", and "example" should be an outer subject of "studying".

If a VERB is allowed to have a `cop` dependent even when it's not a clausal predicate, just a normal adjective-like predicate, then it may require manual effort to distinguish the two situations.

**Comment by nschneid** [comment]
Note that this affects the `nsubj` vs. `nsubj:pass` designation: currently we have

- Great meats that are already cooked

as cop(cooked,are), E:nsubj(cooked,meats), which is confusing—should it be E:nsubj:pass(cooked,meats)? aux:pass instead of cop?

**Comment by amir-zeldes** [comment]
Basically I think we should apply PTB guidelines for VBN/VBG vs. JJ and decide accordingly. These are either compound adjectives or synthetic compounds. Concretely my gut says the passive ones above are really passive:

  * all was mouth-watering - JJ - it is not 'watering the mouth', this is a lexicalized adjective for me
  * ...is mind-blowing - borderline, could go either way on this (can you say "became mind blowing" in the context?) e
  * your cat is stressed out - leaning VBN (could be stressed out by something, though it is intensifiable; I'd negate with not, not un-)
  * It's tailor-made for... - VBN, made by a tailor (at least that's the literal meaning)

**Comment by nschneid** [comment]
And if it's a VERB, "be" should be `aux(:pass)` rather than `cop`?

**Comment by amir-zeldes** [comment]
Yes, I think that would be the consequence. If you want to say something is a predicative adjective and it's *not* passive, then you shouldn't tag it VBN, no?

**Comment by nschneid** [comment]
Yeah, that's why I was surprised to see VERB/cop cases. Maybe the validator is forced to allow such a combination because there could be ellipsis.

**Comment by amir-zeldes** [comment]
It just occurred to me there is an exception: when the VBN/VBG is nominalized, PTB and UD tag based on the morphological category, but syntactically it's now copular, for example:

  * “We are/cop the abandoned/VBN,” the creature said

**Comment by nschneid** [comment]
> It just occurred to me there is an exception: when the VBN/VBG is nominalized, PTB and UD tag based on the morphological category, but syntactically it's now copular, for example:
> 
>   * “We are/cop the abandoned/VBN,” the creature said

Fun, I could see VERB, ADJ, or NOUN as being justified there. I guess you're saying all nonidiomatic participles are tagged as verbs, and being the predicate of a finite clause is not enough to trigger `cop` over `aux` unless there's nominalization as well?

**Comment by amir-zeldes** [comment]
>  I could see VERB, ADJ, or NOUN as being justified there

Yeah, I think one could argue for any of these, but there is an established policy about nominalization (the poor/JJ/ADJ), so I think we're stuck with VERB as the underlying form (from VBN, i.e. the pre-nominalized POS)

> all nonidiomatic participles are tagged as verbs

I don't think the criterion is necessarily idiomaticity, the PTB guidelines have a bunch of tests. The new tokenization sort of changes 'can't be a verb because there's no lemma "well-know"' (for well-known) to 'can't be a verb because adverbial modification' (after post EWT tokenization), and that can conflict with by-phrase etc.

> unless there's nominalization as well?

That one seems to be a valid exception under current guidelines, there may be others as well...

**Comment by nschneid** [comment]
https://github.com/amir-zeldes/gum#182 (discussed offline with @amir-zeldes) discusses allowing passive forms to have copulas when compounded: "self inflicted", "tailor made". This could also be extended to present participles, e.g. "mouth watering", "dull looking".

As copular predicates these are relatively rare. Still, it is an open-ended set. I would favor `ExtPos=ADJ` as an explicit indication that we are treating them as adjectival (hence licensing the copula). Validators looking to flag invalid cop+VERB combinations would then check the ExtPos and not need to maintain a lexical list.

I see that in EWT, if the first word of the compound is a NOUN, it attaches to the participle as `obl:npmod`. Should this be `compound` instead? Queries for all copular participles: [GUM](https://universal.grew.fr/?custom=65d7ea943a681), [EWT](https://universal.grew.fr/?custom=65d7ead558dce)

We might also want to apply `ExtPos=ADJ` to such compounds used attributively, which is the more common case. Other functions are possible too. E.g. in EWT:

- [NOUN+Participle attributive compounds](https://universal.grew.fr/?custom=65d7ebc1d241a)
- [Y+Participle combinations in various functions](https://universal.grew.fr/?custom=65d7f03b77160) where Y attaches to the participle as `amod`, `compound`, or `obl:npmod`
- there are other combinations that *might* be considered compound ADJs, e.g. ADV+Participle as in "well-formed" (idiomatic) or "cheaply made" (non-idiomatic). "This watch is cheaply made" would need permission to have a copula, and `ExtPos=ADJ` could be extended to attributive ADV+Participle combinations too, e.g. "the cheaply made watch". In fact, perhaps any VERB attaching as `amod` (compound or not) automatically qualifies as `ExtPos=ADJ`. [a query that includes advmods](https://universal.grew.fr/?custom=65d7f43b5b87e)
